src/pipeline.ml

```diff
@@ -78,61 +104,75 @@ let or_die = function
 
 (* Pipeline to build the opam base image and the compiler images for a particular architecture. *)
 module Arch = struct
+  let squash = Sys.unix
```

Comment:
Would be good to add a comment saying why we only squash on Linux.

Reply:
Done!

Reply:
2021...!